great-idea/css/index.css

```diff
@@ -63,4 +63,212 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+/*== Theme ===================================================================*/
+
+/*-- Global -----------------------------------*/
+a {
+	text-decoration: none;
+	color: inherit;
+}
+
+/*-- Header -----------------------------------*/
+header.main-navigation a {
+	font-size: smaller;
+	color: #bbb;
+}
+
+/*-- Splash -----------------------------------*/
+.splash .call-to-action h1 {
+	font-size: 80px;
+}
+.button {
+	display: inline-block;
+	border: 1px solid black;
+	padding: 10px 40px;
+	background: white;
+}
+
+/*-- Main Content -----------------------------*/
+.content {
+	margin-top: 50px;
+	border-style: solid;
+	border-color: black;
+	border-width: 1px 0px;
+}
+.container header.main-navigation, .container .splash, .container .blurb-row, .container.page-index footer {
+	padding: 28px 0px;
+}
+
+
+/*== Layout ==================================================================*/
+
+/*-- Global -----------------------------------*/
+
+/*-- Container --------------------------------*/
+.container {
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+}
+.container > header, .container > main, .container.page-index > footer {
+	width: 875px;
+}
+
+/*-- Header -----------------------------------*/
+header.main-navigation {
+	display: flex;
```

Comment:
I wanted to avoid setting a height for two reasons:

1) It would require setting height on each marketing blurb, which means that changing the text in each paragraph could break the layout. (Not a problem here, but trying to think forward to future projects.)

2) Setting the height seems like it would undermine the value in using flex & wrap. I use the fact that height is flexible in the responsive stretch goal.

How would you resolve the above two concerns without padding?